backends/dpdk/spec.cpp

```diff
@@ -449,13 +460,16 @@ std::ostream& IR::DpdkLearner::toSpec(std::ostream& out) const {
     } else {
         out << "\tsize " << DEFAULT_LEARNER_TABLE_SIZE << std::endl;
     }
-    if (auto size = properties->getProperty("psa_idle_timeout")) {
-        out << "\ttimeout " << DPDK::toStr(size->value) << "" << std::endl;
-    } else {
-        out << "\ttimeout " << DEFAULT_LEARNER_TABLE_TIMEOUT << std::endl;
-    }
 
-    out << "}" << std::endl;
+    // The timeout values are currently hardcoded here
+    // FIXME: This should come from P4 program but currently we don't have a way
```

Comment:
This is decided to be a some default hardcoded value. The values and number of values in the timeout structure are yet to be confirmed. Will remove this FIXME once Cristian confirms the default vaues

Reply:
This is updated after confirmation from Cristian.